core/sqf/sqenvcom.sh

```diff
@@ -580,6 +547,13 @@ EOF
     echo "**** ERROR: Unable to determine location of HBase lib directory"
   fi
 
+  if [[ -d $TOOLSDIR/thrift-0.9.0 ]]; then
+    # this is mostly for a build environment, where we need
+    # thrift from TOOLSDIR
+    export THRIFT_LIB_DIR=$TOOLSDIR/thrift-0.9.0/lib
+    export THRIFT_INC_DIR=$TOOLSDIR/thrift-0.9.0/include
+  fi
+
   if [ -n "$HBASE_CNF_DIR" -a -n "$HADOOP_CNF_DIR" -a \
```

Comment:
Good point, I should not have removed it from bldenvchk (did that when I tried to eliminate Thrift altogether - unsuccessfully). I'll put the check back into bldenvchk, because the build would fail.